pkgs/os-specific/linux/tuxedo-drivers/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, fetchFromGitLab, kernel, linuxHeaders, pahole }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tuxedo-drivers-${kernel.version}";
+  version = "4.3.0";
+
+  src = fetchFromGitLab {
+    owner = "tuxedocomputers/development/packages";
+    repo = "tuxedo-drivers";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-IZ1K7n4OrhI52u7EH0ljVkKwc/uf53sMd9qpTLjvzls=";
+  };
+
+  buildInputs = [
+    pahole
+    linuxHeaders
+  ];
+
+  makeFlags = [ "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build" ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/lib/modules/${kernel.modDirVersion}"
+
+    # We have a bigger number of kernel modules in this project, doesnt make sense to list them manually.
+    for module in `find . -name "*.ko" -type f`; do
+        mv $module $out/lib/modules/${kernel.modDirVersion}
+    done
```

Comment:
Does `make install` not do the right thing? From reading the upstream Makefile it should. Maybe with some help from `installFlags = [ "INSTALL_MOD_PATH=${placeholder "out"}" ];`

Reply:
You are right! That is exactly what I did in my other kernel module commit. ([pkgs](https://github.com/NixOS/nixpkgs/tree/master/pkgs)/[os-specific](https://github.com/NixOS/nixpkgs/tree/master/pkgs/os-specific)/[linux](https://github.com/NixOS/nixpkgs/tree/master/pkgs/os-specific/linux)/[trelay](https://github.com/NixOS/nixpkgs/tree/master/pkgs/os-specific/linux/trelay))

I just mindlessly transfered the file from tuxedo-keyboard, dont know why they did it differently there tbh!